library/Rules/PostalCode.php

```diff
@@ -113,7 +113,7 @@ class PostalCode extends Regex
         "NF" => "/^(\d{4})$/",
         "NG" => "/^(\d{6})$/",
         "NI" => "/^(\d{7})$/",
-        "NL" => "/^(\d{4}[A-Z]{2})$/",
+        'NL' => "/^(\A[1-9]\d{3} ?[a-zA-Z]{2}\z)$/",
```

Comment:
Hey @HDsign, do you have any news on it? At least the company I work in The Netherlands we don't allow lower case chars.